scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/GradleBuildTool.scala

```diff
@@ -49,70 +48,33 @@ class GradleBuildTool(index: IndexCommand) extends BuildTool("Gradle", index) {
         "gradle"
 
     TemporaryFiles.withDirectory(index) { tmp =>
-      val toolchains = GradleJavaToolchains
-        .fromWorkspace(this, index, gradleCommand, tmp)
-      toolchains.gradleVersion match {
-        case Some(gradleVersion)
-            if gradleVersion.startsWith("6.7") &&
-              toolchains.toolchains.nonEmpty =>
-          index
-            .app
-            .error(
-              "scip-java does not support Gradle 6.7 when used together with Java toolchains. " +
-                "To fix this problem, upgrade to Gradle version 6.8 or newer and try again."
-            )
-          CommandResult(1, Nil)
-        case _ =>
-          runCompileCommand(toolchains)
-      }
+      runCompileCommand(tmp, gradleCommand)
     }
   }
 
   private def runCompileCommand(
-      toolchains: GradleJavaToolchains
+      // toolchains: GradleJavaToolchains,
+      tmp: Path,
+      gradleCommand: String
   ): CommandResult = {
-    val script = initScript(toolchains, toolchains.tmp).toString
+    val script = initScript(tmp).toString
     val buildCommand = ListBuffer.empty[String]
-    buildCommand += toolchains.gradleCommand
+    buildCommand += gradleCommand
     buildCommand += "--no-daemon"
     buildCommand += "--init-script"
     buildCommand += script
-    if (toolchains.toolchains.nonEmpty) {
-      buildCommand += "-Porg.gradle.java.installations.auto-detect=false"
-      buildCommand += "-Porg.gradle.java.installations.auto-download=false"
-      buildCommand +=
-        s"-Porg.gradle.java.installations.paths=${toolchains.paths()}"
-    }
     buildCommand += "-Pkotlin.compiler.execution.strategy=in-process"
+    buildCommand += s"-Dsemanticdb.targetroot=$targetroot"
     buildCommand ++=
       index.finalBuildCommand(
-        List[Option[String]](
-          Some("clean"),
-          if (toolchains.isJavaEnabled)
-            Some("compileTestJava")
-          else
-            None,
-          if (toolchains.isScalaEnabled)
-            Some("compileTestScala")
-          else
-            None,
-          if (toolchains.isKotlinEnabled)
-            Some("compileTestKotlin")
-          else
-            None,
-          if (toolchains.isKotlinMultiplatformEnabled)
-            Some("compileTestKotlinJvm")
-          else
-            None
-        ).flatten
+        List("clean", "scipCompileAll", "scipPrintDependencies")
```

Comment:
What happens with `scipPrintDependencies` if `scipCompileAll` partially fails? Would it make sense to run these commands in reverse order?